frontend/apps/web/package.json

```diff
@@ -65,6 +65,7 @@
     "react-icons": "^5.2.1",
     "recharts": "^2.12.7",
     "sonner": "^1.5.0",
+    "sql-formatter": "^15.4.2",
```

Comment:
One caveat here is this is platform specific. So if using docker dev, let docker do `npm install` otherwise the runs page will not load.